cargo-verify/src/main.rs

```diff
@@ -438,7 +446,23 @@ fn verifier_run(opt: &Opt, bcfile: &Path, name: &str, entry: &str) -> Status {
         Status::Unknown
     });
 
-    println!("test {} ... {:#}", name, status);
+    if opt.quiet {
+        let s = match status {
+            Status::Unknown => Colour::Yellow.paint("?"),
+            Status::Verified => Colour::Green.paint("."),
+            Status::Error => Colour::Red.paint("F"),
```

Comment:
Maybe E instead of F?

Reply:
I chose F and . (and the colours) to match cargo-test.